app/components/istag-select/istag-select.html

```diff
@@ -1,11 +1,11 @@
 <ng-form name="istagForm">
-  <fieldset ng-disabled="selectDisabled">
+  <fieldset ng-disabled="$ctrl.selectDisabled">
     <div class="row">
       <div class="form-group col-sm-4">
         <label class="sr-only">Namespace</label>
-        <ui-select ng-required="selectRequired" ng-model="istag.namespace" ng-disabled="selectDisabled" ng-change="istag.imageStream = null; istag.tagObject = null;" append-to-body="appendToBody">
+        <ui-select ng-required="$ctrl.selectRequired" ng-model="$ctrl.istag.namespace" ng-disabled="$ctrl.selectDisabled" ng-change="$ctrl.istag.imageStream = null;$ctrl.istag.tagObject = null;$ctrl.namespaceChanged($ctrl.istag.namespace)" append-to-body="appendToBody">
```

Comment:
It would be better to make this just

```html
ng-change="$ctrl.namespaceChanged($ctrl.istag.namespace)"
```

and null the other values in the namespaceChanged() function in the controller.